sdk/ai/azure-ai-assistants/samples/async_samples/sample_assistant_stream_eventhandler_async.py

```diff
@@ -114,14 +114,13 @@ async def sample_assistant_stream_iteration():
         ) as stream:
             await stream.until_done()
 
-        messages = await assistant_client.list_messages(thread_id=thread.id)
-        logging.info(f"Messages: {messages}")
-
         await assistant_client.delete_assistant(assistant.id)
         logging.info("Deleted assistant")
 
+        messages = await assistant_client.list_messages(thread_id=thread.id)
```

Comment:
was there a reason to change the order? just curious

Reply:
I found to read the message in terminal, I need to scroll up to skip lot of content.   So I move the message down.